dtfs-central-api/src/v1/routes/middleware/payload-validation/validate-put-portal-facility-amendment-payload.test.ts

```diff
@@ -0,0 +1,294 @@
+import { ObjectId } from 'mongodb';
+import { createMocks } from 'node-mocks-http';
+import { HttpStatusCode } from 'axios';
+import { getUnixTime } from 'date-fns';
+import { AUDIT_USER_TYPES, CURRENCY } from '@ukef/dtfs2-common';
+import { generatePortalAuditDetails } from '@ukef/dtfs2-common/change-stream';
+import { aPortalUser } from '../../../../../test-helpers';
+import { validatePutPortalFacilityAmendmentPayload } from './validate-put-portal-facility-amendment-payload';
+
+const validDealId = new ObjectId().toString();
+const validAuditDetails = generatePortalAuditDetails(aPortalUser()._id);
+const validAmendment = {
```

Comment:
Do we have duplication of mock amendments for testing purposes? If so I think we should mock into the test mocks directory

Reply:
I've created two mock functions in libs common to generate this :)